runtime_config_linux.go

```diff
@@ -195,7 +195,7 @@ type Network struct {
 	// Set class identifier for container's network packets
 	ClassID *uint32 `json:"classID"`
 	// Set priority of network traffic for container
-	Priorities []InterfacePriority `json:"priorities"`
+	Priorities []*InterfacePriority `json:"priorities"`
```

Comment:
In general, we don't need pointers for slices and maps. I can go cleanup
the Spec again once I get some spare cycles.

On Mon, Jan 25, 2016 at 4:46 PM, Mrunal Patel <notifications@github.com>
wrote:

> In runtime_config_linux.go
> <https://github.com/opencontainers/specs/pull/316#discussion_r50781540>:
>
> > @@ -195,7 +195,7 @@ type Network struct {
> >  	// Set class identifier for container's network packets
> >  	ClassID *uint32 `json:"classID"`
> >  	// Set priority of network traffic for container
> > -	Priorities []InterfacePriority `json:"priorities"`
> > +	Priorities []*InterfacePriority `json:"priorities"`
>
> @vishh <https://github.com/vishh> I am fine one way or the other. But we
> have them as pointers in more places compared to where we don't.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/opencontainers/specs/pull/316/files#r50781540>.
>